vrt/config.js

```diff
@@ -13,7 +13,7 @@ const config = {
     interactions: [
       {
         name: 'openedMenu',
-        behavior: async page => {
+        behavior: async (page) => {
```

Comment:
Are all of these `page` -> `(page)` changes intentional?

Reply:
Sorry, the editor changes it when auto save and I don't notice it .